types/ej.web.all/index.d.ts

```diff
@@ -3271,79 +3263,67 @@ export interface Model {
 	targetID?: string;
 
 	/** Triggers before the AJAX request begins to load data in the ListBox widget. */
-	actionBegin?(e: ActionBeginEventArgs): void;
+	actionBegin? (e: ActionBeginEventArgs): void;
 
 	/** Triggers after the data requested via AJAX is successfully loaded in the ListBox widget. */
-	actionSuccess?(e: ActionSuccessEventArgs): void;
+	actionSuccess? (e: ActionSuccessEventArgs): void;
 
 	/** Triggers when the AJAX requests complete. The request may get failed or succeed. */
-	actionComplete?(e: ActionCompleteEventArgs): void;
+	actionComplete? (e: ActionCompleteEventArgs): void;
 
 	/** Triggers when the data requested from AJAX get failed. */
-	actionFailure?(e: ActionFailureEventArgs): void;
+	actionFailure? (e: ActionFailureEventArgs): void;
 
 	/** Event will be triggered before the requested data via AJAX once loaded in successfully. */
-	actionBeforeSuccess?(e: ActionBeforeSuccessEventArgs): void;
+	actionBeforeSuccess? (e: ActionBeforeSuccessEventArgs): void;
 
 	/** Triggers when the item selection is changed. */
-	change?(e: ChangeEventArgs): void;
+	change? (e: ChangeEventArgs): void;
 
 	/** Triggers when the list item is checked or unchecked. */
-	checkChange?(e: CheckChangeEventArgs): void;
+	checkChange? (e: CheckChangeEventArgs): void;
 
 	/** Triggers when the ListBox widget is created successfully. */
-	create?(e: CreateEventArgs): void;
+	create? (e: CreateEventArgs): void;
 
 	/** Triggers when the ListBox widget is destroyed successfully. */
-	destroy?(e: DestroyEventArgs): void;
+	destroy? (e: DestroyEventArgs): void;
 
 	/** Triggers when focus the listbox items. */
-	focusIn?(e: FocusInEventArgs): void;
+	focusIn? (e: FocusInEventArgs): void;
 
 	/** Triggers when focus out from listbox items. */
-	focusOut?(e: FocusOutEventArgs): void;
+	focusOut? (e: FocusOutEventArgs): void;
 
 	/** Triggers when the list item is being dragged. */
-	itemDrag?(e: ItemDragEventArgs): void;
+	itemDrag? (e: ItemDragEventArgs): void;
 
 	/** Triggers when the list item is ready to be dragged. */
-	itemDragStart?(e: ItemDragStartEventArgs): void;
+	itemDragStart? (e: ItemDragStartEventArgs): void;
 
 	/** Triggers when the list item stops dragging. */
-	itemDragStop?(e: ItemDragStopEventArgs): void;
+	itemDragStop? (e: ItemDragStopEventArgs): void;
 
 	/** Triggers when the list item is dropped. */
-	itemDrop?(e: ItemDropEventArgs): void;
+	itemDrop? (e: ItemDropEventArgs): void;
 
 	/** Triggers when a list item gets selected. */
-	select?(e: SelectEventArgs): void;
+	select? (e: SelectEventArgs): void;
 
 	/** Triggers when a list item gets unselected. */
-	unselect?(e: UnselectEventArgs): void;
+	unselect? (e: UnselectEventArgs): void;
 }
 
 export interface ActionBeginEventArgs {
```

Comment:
An empty interface is indistiguishable from other empty interfaces, including `{}`. If there are no properties, then probably the interface should be deleted.